app/models/dog_searcher.rb

```diff
@@ -71,7 +71,9 @@ def self.search(params: {}, manager: false)
   private
 
   def tracking_id_search?
```

Comment:
I'll try and do a query of the production database tonight and see what the Microchip IDs actually look like.   I know several brands are in use and I kind of forget what the formats look like.  There is a chance that some are a blend of letters and numbers.